Els_kom_Core/interfaces/ICallbackPlugin.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2014-2018, Els_kom org.
+// https://github.com/Elskom/
+// All rights reserved.
+// license: MIT, see LICENSE for more details.
+
+/* 0x01050000 = v1.5.0.0 */
```

Comment:
I think the callback plugins should be in 1.4.9.8 for those who want to get a kom spoofing plugin for Elsword or Grand Chase sooner than 1.5.0.0.